.github/workflows/flake8.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: Flake8
+on: pull_request
+jobs:
+  flake8:
+    name: Check code with Flake8
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Setup environment
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8
+      - name: Run Flake8
+        # run: flake8 . --ignore=E266,E261,E265,W503 --max-line-length=120 --count
+        run: flake8 postprocess --ignore=E266,E261,E265,W503 --max-line-length=120 --count # only do postprocessing code
```

Comment:
We already have Python files in the repo, maybe it's worth running it for all of the directories?

Reply:
Linter appeased and enabled.